src/cli/commands/files/add.js

```diff
@@ -94,26 +107,62 @@ module.exports = {
       }
     }
 
-    createAddStream((err, addStream) => {
-      if (err) {
+    // if there are piped arguments, input is data to publish instead of a file
+    // path
+    if(hasPipedArgs) {
+      const data = argv.file
+      const dataStream = new stream.Readable()
+      dataStream.push(Buffer.from(data, 'utf8'))
+      dataStream.push(null)
+      createAddStream((err, addStream) => {
+        if(err)
         throw err
-      }
+        
+        addDataPipeline([dataStream], addStream)
+      })
```

Comment:
Something doesn't look right here:

`createAddStream` is going to glob a dir and return the stream which has all the files resulting from that glob.

If you want to add one stream, you should not need to call that.